sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -553,19 +594,29 @@ AllocaCommandBase *Scheduler::GraphBuilder::getOrCreateAllocaForReq(
 
   Requirement FullReq(/*Offset*/ {0, 0, 0}, Req->MMemoryRange,
```

Comment:
Could you check if this is still needed?

Reply:
Yes, there is still need in `FullReq`.